### ISSUE TITLE
No module named 'leak.version_parser'

### ISSUE THREAD

**Issue by jonafato** [bmwant/leak#1]
When trying to run `leak`, I get the following error:

```
$ leak leak
Traceback (most recent call last):
  File "/Users/jonafato/.virtualenvs/tmp-98d5d829c65a0c/bin/leak", line 2, in <module>
    import leak
  File "/Users/jonafato/.virtualenvs/tmp-98d5d829c65a0c/lib/python3.5/site-packages/leak/__init__.py", line 1, in <module>
    from .main import cli
  File "/Users/jonafato/.virtualenvs/tmp-98d5d829c65a0c/lib/python3.5/site-packages/leak/main.py", line 13, in <module>
    from .version_parser import versions_split
ImportError: No module named 'leak.version_parser'
```

Is the `version_parser` module missing from the distribution and git repository?

**Comment by AhnSeongHyun** [comment]
me too, raise this error.

**Comment by jqtrde** [comment]
Same situation here as well.

**Comment by bmwant** [comment]
@AhnSeongHyun @jacquestardie @jonafato. Fixed in new version 1.0.4

**Comment by jonafato** [comment]
Working perfectly now, thanks!
